lib/actions/request_review.js

```diff
@@ -1,10 +1,27 @@
 const { Action } = require('./action')
+const logger = require('../logger')
 const _ = require('lodash')
 
-const createRequestReview = async (context, number, reviewers) => {
-  return context.octokit.pulls.requestReviewers(
-    context.repo({ pull_number: number, reviewers })
-  )
+const createRequestReview = async (context, number, reviewers, actionObj) => {
+  let res
+  try {
+    res = await context.octokit.pulls.requestReviewers(
+      context.repo({ pull_number: number, reviewers })
+    )
+  } catch (err) {
+    // skip when error is "Review can not be requested from pull request author"
+    if (!err.message.toLowerCase().includes('pull request author')) {
```

Comment:
This seem redundant, since the author is being filtered out.

Reply:
good point, removed